.eslintrc.js

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why this is working now? @AuraAlba

Reply:
I added this file in the list of eslintignore so this line is not necessary